web/themes/custom/novel/templates/layout/node--page--full.html.twig

```diff
@@ -1,3 +1,14 @@
 <article>
-    {{ content.field_paragraphs }}
+  {% if node.hasField('field_display_titles') and
+        node.get('field_display_titles').getString() == '1' %}
+    <header class="article-header">
+      <h1 class="article-header__title">
+      {{ content.field_hero_title.0|default(label) }}
+      </h1>
+
+      {{ content.field_subtitle }}
+    </header>
+  {% endif %}
```

Comment:
Please consider the approach with rendering the title conditionally.

Accessibility requires us to have a proper tree of heading levels. If we do not render the page title as a `<h1>` we need some other component (paragraph?) to have that `<h1>` heading. We do not support this at the moment and in my opinion we do not want to add that option/configurability to Paragraphs either.

If we do not want to display the title then I suggest we render it but [hide it visually using CSS](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/6e5f558e015c59f3f54bc632b9031d0c24f29a6c/src/styles/scss/tools/mixins.misc.scss#L1).

Reply:
Fair enough, I've updated it now:

https://github.com/danskernesdigitalebibliotek/dpl-cms/compare/5180e26e55e26b3137744919ca38a2b8d316e8ab..51bf549a32a7966ca8122c1d84f57a2b5fd1a685